canonical-websites/docs.ubuntu.com
# Update to latest run script, and move docs building into package.json script

- Update to ./run from generator@v2.1.1
- Add docs building to the package.json commands

Fixes #95 

## QA

``` bash
./run clean
./run  # This will take a long time the first time, but should work
```

Check the site works as expected.

``` bash
./run watch  # Check this still works, fairly quickly.
./run build  # Shouldn't take so long the second time around
```
